src/packagedcode/jar_manifest.py

```diff
@@ -369,10 +339,61 @@ def dget(s):
     # Deps:
     # Require-Bundle
 
-    package['notes'] = dget('Comment')
+    #package['notes'] = dget('Comment')
     return package
 
 
+def parse_scm_connection(scm_connection):
+    """
+    Return an SPDX vcs_url given a Maven `scm_connection` string or the string
+    as-is if it cannot be parsed.
+
+    See https://maven.apache.org/scm/scm-url-format.html
+        scm:<scm_provider><delimiter><provider_specific_part>
+
+    scm:git:git://server_name[:port]/path_to_repository
+    scm:git:http://server_name[:port]/path_to_repository
+    scm:git:https://server_name[:port]/path_to_repository
+    scm:git:ssh://server_name[:port]/path_to_repository
+    scm:git:file://[hostname]/path_to_repository
+    """
+
+    delimiter = '|' if '|' in scm_connection else ':'
+    segments = scm_connection.split(delimiter, 2)
+    if not len(segments) == 3:
+        # we cannot parse this so we return it as is
+        return scm_connection
+
+    _scm, scm_tool, vcs_url = segments
+    # TODO: vcs_tool is not yet supported
+    normalized = normalize_vcs_url(vcs_url, vcs_tool=scm_tool)
+    if normalized:
+        vcs_url = normalized
+
+    if not vcs_url.startswith(VCS_URLS):
+        if not vcs_url.startswith(scm_tool):
+            vcs_url = '{scm_tool}+{vcs_url}'.format(**locals())
+
+    return vcs_url
+
+
+def get_datasource_id(package_type):
+    """
+    Get the corresponding `datasource_id` for the given
+    `package_type`. This is a seperate function to avoid
+    cyclic imports.
+    """
+    from packagedcode.maven import JavaJarManifestHandler
+    from packagedcode.maven import JavaOSGiManifestHandler
+
+    if package_type == 'maven':
```

Comment:
```suggestion
    if package_type in ['maven', 'jar']:
        return JavaJarManifestHandler.datasource_id
```

Minor change just to avoid repetition.

Reply:
Added this, thanks!